packages/fdc3-schema/schemas/api/channelChangedEvent.schema.json

```diff
@@ -42,6 +42,14 @@
 							"type": "null"
 						}
 					]
+				},
+				"userChannels": {
```

Comment:
I don't think this should be here - the event should not have to carry all the user channel definitions, which are supposed to remain constant (there are no event s for changes in the user channel definitions which are supposed to remain fixed). These should already have been retrieved by the client code and hence this duplication is inefficient.

At most the definition of the channel being selected could be included - but even that should not be necessary.

Also this change to the message schema will need to be approved by the SWG, where a change to the implementation only can be approved by the maintainers.